src/aviary/dataset_server.py

```diff
@@ -7,17 +7,17 @@
 from itertools import starmap
 from typing import Generic, TypeVar
 
-import uvicorn
 from pydantic import BaseModel, Field
 
 from aviary.env import Environment, TaskDataset
 from aviary.tools import MessagesAdapter, ToolRequestMessage, ToolsAdapter
 
 try:
+    import uvicorn
     from fastapi import FastAPI, HTTPException
 except ImportError:
     # We will raise if a TaskDatasetServer is instantiated but fastapi is not available
-    FastAPI = HTTPException = None  # type: ignore[misc,assignment]
+    uvicorn = FastAPI = HTTPException = None  # type: ignore[misc,assignment]
```

Comment:
Can we update the `ImportError` below and the comment above to mention `uvicorn`

Reply:
Done! Also realized I had the wrong pypi package name